californium-core/src/main/java/org/eclipse/californium/core/coap/LinkFormat.java

```diff
@@ -51,6 +51,7 @@ public class LinkFormat {
 	public static final String TITLE = "title";
 	public static final String OBSERVABLE = "obs";
 	public static final String LINK = "href";
+	public static final String OSCOREONLY = "osc";
```

Comment:
Please add 

```
// RFC 8613
```

Reply:
Added in commit 4281708a39346675a1d6ee0f88f26bab0c902280